lib/savings_screen/view/savings_page.dart

```diff
@@ -0,0 +1,362 @@
+                            children: [
+                              Text(
+                                'Mortgage',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .displaySmall
+                                    ?.copyWith(
+                                      color: AppColors.secondaryColorW,
+                                      fontWeight: FontWeight.w600,
+                                      fontFamily: 'open sans',
+                                      fontSize: 14,
+                                    ),
+                              ),
+                              Text(
+                                r'$ 22,500.50',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .displaySmall
+                                    ?.copyWith(
+                                      color: AppColors.secondaryColorW,
+                                      fontWeight: FontWeight.w600,
+                                      fontFamily: 'open sans',
+                                      fontSize: 16,
+                                    ),
+                              ),
+                            ],
+                          ),
+                        ],
+                      ),
+                      const SizedBox(
+                        height: 20,
+                      ),
+                      const Divider(
+                        height: 0.5,
+                        color: Color(0xffE7E7E7),
+                        thickness: 0.5,
+                      ),
+                      const SizedBox(
+                        height: 10,
+                      ),
+                      Column(
+                        // ignore: avoid_redundant_argument_values
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                            children: [
+                              Text(
+                                'New Drone',
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .displaySmall
+                                    ?.copyWith(
+                                      color: AppColors.secondaryColorW,
+                                      fontWeight: FontWeight.w400,
+                                      fontFamily: 'open sans',
+                                      fontSize: 14,
+                                    ),
+                              ),
+                              Text(
+                                r'$ 79.45',
+                                style: Theme.of(context)
```

Comment:
This theme extension seems to be very repetitive, you could extract it to a variable and use that variable so if it has to change in the future, you can just change it in the level of the variable and it will effect the change everywhere